cucumber-messages/specs/TestRunStarted.feature

```diff
@@ -0,0 +1,22 @@
+Feature: Sending TestRunStarted messages
```

Comment:
How is this feature file processed? I don't see anything running it in this repo.

I'm not convinced Cucumber is the right tool to test the messages. I'm not even sure the messages need to be "tested" as they have no logic, they are just structs.